src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2

```diff
@@ -2,32 +2,66 @@ import json
 import os
 import streamlit as st
 from pathlib import Path
+from streamlit_quill import st_quill
+from bs4 import BeautifulSoup, NavigableString, Tag
 
 from promptflow._sdk._utils import print_yellow_warning
 from promptflow._sdk._serving.flow_invoker import FlowInvoker
 
-
+IMAGE_KEY = "data:image/png:base64"
```

Comment:
data:image/png;base64, should be ; instead of :

Reply:
fixed